free-2.md

```diff
@@ -58,12 +84,15 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+If a suitable TTS API/Library can be found, the majority of time (maybe 70%+) would be spent directly engaging in the language aspects of the project.  A "systems"-type aspect that would still need to be addressed is the I/O for the audio.  Clips generated from the TTS API/Library would have to be properly stitched together and then output as a single audio file.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+The project seems decently-sized.  The scope is very specific, meaning that there aren't a huge number of features to be implemented.
```

Comment:
Yeah, like you said above, I think it depends pretty largely on how good TTS api's are.